R/molecule.R

```diff
@@ -0,0 +1,46 @@
+#' @title Molecule
+#'
+#' @description A molecule defined in a compartment of the system
+#'
+#' @details  Derived from \link{Quantity}, please see base class documentation.
+#'
+#' @docType class
+#' @name Molecule
+#'
+#' @keywords Molecule
+#' @format NULL
+Molecule <- R6::R6Class(
+  "Molecule",
+  inherit = Quantity,
+  active = list(
+    value = function(value) {

```

Comment:
@PavelBal @Yuri05 
Molecules are a different beast. Depending on the project type (amount or concentration) the value is either a constant or a formula But the real start values in for the second case saved in a parameter called Start value. I don't want the user to have to think about it so I basically save a reference to the paraemter start value if if exists and use this parameter to set and retrieve the value. If it does not exist, well it;s a Molecule in amount that can be used AS IS